internal/manifests/collector/ingress.go

```diff
@@ -78,7 +87,7 @@ func createPathIngressRules(otelcol string, hostname string, ports []corev1.Serv
 	for i, port := range ports {
 		portName := naming.PortName(port.Name, port.Port)
 		paths[i] = networkingv1.HTTPIngressPath{
-			Path:     "/" + port.Name,
+			Path:     "/" + port.Name + "(/|$)(.*)",
```

Comment:
Can you explain why the path changes? 

This will also result in a breaking change.

Reply:
The traces, for instance, are reported to `<<URL>>/v1/traces`. Without this, you can only access to `<<URL>>`. Everything you send to `<URL>/<subpath>` will be redirected to `<URL>`. Exposing the collector doesn't work right now.

Reply:
this fix will only work for nginx right? for anything using a default ingress provider this will have no effect? @pavolloffay this technically isn't a breaking change because we choose prefix currently, so anything after `/<port-name>` is already accepted and this regex doesn't change that because the current options are captured within the regex. ex:
```
/<port-name> is matched by /<port-name>(/|$)(.*)
/<port-name>/somethingelse is matched by /<port-name>(/|$)(.*)
```